labseu2-workout-tracker/client
# History to redux

##  What does this PR do
Workouts history moved to the redux store.


##  Description of task to be completed

Sessions history action created,
Workouts action created.
Now the state is hosted in redux store.

##  How should this manually be tested
Yarn install, make sure you have a session in DB.
If you don't have just add one.

##  What are the relevant Trello board stories
https://trello.com/c/k4RRcCDv/195-connecting-history-comp-state-into-redux

##  Screenshots (If appropriate)


##  Questions (If any):
